src/Uno.UI/UI/Xaml/Controls/Primitives/ButtonBase/ButtonBase.mux.cs

```diff
@@ -0,0 +1,737 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// ButtonBase_Partial.h, ButtonBase_Partial.cpp
+
+#nullable enable
+
+using System;
+using System.Windows.Input;
+using Uno.Disposables;
+using Uno.UI.Xaml.Core;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Input;
+
+namespace Windows.UI.Xaml.Controls.Primitives
+{
+	public partial class ButtonBase
+	{
+		// Last known position of the pointer with respect to this button.
+		// Declared as protected so the RepeatButton can access it.
+		protected Point _pointerPosition = Point.Zero;
```

Comment:
Didn't find it's usage in the `RepeatButton`. If expected, and unless this code is coming from WinUI, I would remove that private field which is only set and read immediately after in the `OnPointerMoved`.